src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -347,7 +347,7 @@ private void testCancelAddingNewQuestion() {
 
         feedbackEditPage.clickNewQuestionButton();
         feedbackEditPage.selectNewQuestionType("NUMSCALE");
-        assertTrue(feedbackEditPage.isAllFeedbackPathOptionsEnabled());
+        assertTrue(feedbackEditPage.isAllFeedbackPathOptionsEnabledForNewQuestion());
```

Comment:
Okay yep that answers my question, I was just wondering why the scope of this method was so specific. I'm guessing it'll be fixed with #5802 then.